### PR TITLE
v08 spec alignment - Support custom function type definition

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <filippespolti@gmail.com>

**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:

**Special notes for reviewers**:

**Additional information (if needed):**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,6 +19,7 @@ require (
 	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/leodido/go-urn v1.2.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/serverlessworkflow/sdk-go v1.0.0 // indirect
```
Comment (lsytj0413) [inline go.mod:22]: why import v1？

File: model/function.go [hunk 2]
```diff
@@ -29,6 +29,8 @@ const (
 	FunctionTypeAsyncAPI FunctionType = "asyncapi"
 	// FunctionTypeOData ...
 	FunctionTypeOData FunctionType = "odata"
+	// FunctionTypeCustom ...
```
Comment (lsytj0413) [inline model/function.go:32]: we should complete the comment, such as `FunctionTypeCustom is the additional function types that extend the ones defined in the specification which implement by runtime`

File: go.mod (hunk 1 above)
Comment (ricardozanini) [reply go.mod:22]: This is happening in his branch, I made the same comment in the other PR.

File: model/function.go (hunk 2 above)
Comment (ricardozanini) [reply model/function.go:32]: @spolti please add the description from the scheme.

**Review by ricardozanini — APPROVED** [review]
Fine after the comments are handled.

File: go.mod (hunk 1 above)
Comment (spolti) [reply go.mod:22]: oh, that again. lol

**Pull request merged**